src/ApiException.php

```diff
@@ -263,11 +282,51 @@ private static function create(
             $metadata = iterator_to_array($metadata);
         }
 
-        return new ApiException($message, $rpcCode, $rpcStatus, [
-            'previous' => $previous,
-            'metadata' => $metadata,
-            'basicMessage' => $basicMessage,
-        ]);
+        return new ApiException(
+            $message,
+            $rpcCode,
+            $rpcStatus,
+            [
+                'previous' => $previous,
+                'metadata' => $metadata,
+                'basicMessage' => $basicMessage,
+            ],
+            $protobufErrors ?? []
+        );
+    }
+
+    /**
+     * Encodes decoded metadata to the Protobuf error type
```

Comment:
(not familiar with the code base) With `rest` transport, the error is returned from the server as the body of the response. So that does it mean that you're taking it from "metadata"?